docs/content/docs/tutorials/aurora-serverless-v2.md

```diff
@@ -0,0 +1,313 @@
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBSubnetGroup
+metadata:
+ name: ${RDS_SUBNET_GROUP_NAME}
+ namespace: ${APP_NAMESPACE}
+spec:
+ name: ${RDS_SUBNET_GROUP_NAME}
+ description: ${RDS_SUBNET_GROUP_DESCRIPTION}
+ subnetIDs:
+$(printf " - %s\n" ${EKS_SUBNET_IDS})
+ tags: []
+EOF
+
+kubectl apply -f db-subnet-groups.yaml
+
+RDS_SECURITY_GROUP_NAME="ackSecurityGroup"
+RDS_SECURITY_GROUP_DESCRIPTION="ACK security group"
+
+EKS_CIDR_RANGE=$(aws ec2 describe-vpcs \
+ --vpc-ids "${EKS_VPC_ID}" \
+ --query "Vpcs[].CidrBlock" \
+ --output text
+)
+
+RDS_SECURITY_GROUP_ID=$(aws ec2 create-security-group \
+ --group-name "${RDS_SUBNET_GROUP_NAME}" \
+ --description "${RDS_SUBNET_GROUP_DESCRIPTION}" \
+ --vpc-id "${EKS_VPC_ID}" \
+ --output text
+)
+aws ec2 authorize-security-group-ingress \
+ --group-id "${RDS_SECURITY_GROUP_ID}" \
+ --protocol tcp \
+ --port 5432 \
+ --cidr "${EKS_CIDR_RANGE}"
+```
+
+Set up a master password using a Kubernetes Secret. SET `RDS_DB_USERNAME` and `RDS_DB_PASSWORD` to your preferred values for your RDS credentials:
+
+```bash
+export APP_NAMESPACE=mydb
+kubectl create ns "${APP_NAMESPACE}"
+
+RDS_DB_USERNAME="adminer"
+RDS_DB_PASSWORD="password"
+
+kubectl create secret generic -n "${APP_NAMESPACE}" ack-creds \
+  --from-literal=username="${RDS_DB_USERNAME}" \
+  --from-literal=password="${RDS_DB_PASSWORD}"
+```
+
+You can now create an Aurora Serverless v2 cluster:
+
+```bash
+export AURORA_DB_CLUSTER_NAME="ack-db"
+export AURORA_DB_INSTANCE_NAME="ack-db-instance01"
+export AURORA_DB_INSTANCE_CLASS="db.serverless"
+export MAX_ACU=64
+export MIN_ACU=4
+
+cat <<-EOF > asv2-db-cluster.yaml
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBCluster
+metadata:
+  name: asv2-db
+  namespace: ${APP_NAMESPACE}
+spec:
+  backupRetentionPeriod: 7
+  serverlessV2ScalingConfiguration:
+    maxCapacity: ${MAX_ACU}
+    minCapacity: ${MIN_ACU}
+  dbClusterIdentifier: ${AURORA_DB_CLUSTER_NAME}
+  dbSubnetGroupName: ${RDS_SUBNET_GROUP_NAME}
+  engine: aurora-postgresql
+  engineVersion: "13.6"
+  masterUsername: adminer
+  masterUserPassword:
+    namespace: ${APP_NAMESPACE}
+    name: ack-creds
+    key: password
+  vpcSecurityGroupIDs:
+     - ${RDS_SECURITY_GROUP_ID}
+EOF
+
+kubectl apply -f asv2-db-cluster.yaml
+
+
+cat <<-EOF > asv2-db-instance.yaml
+apiVersion: rds.services.k8s.aws/v1alpha1
+kind: DBInstance
+metadata:
+  name: asv2-db-instance
+  namespace: ${APP_NAMESPACE}
+spec:
+  dbInstanceClass: ${AURORA_DB_INSTANCE_CLASS}
+  dbInstanceIdentifier: ${AURORA_DB_INSTANCE_NAME}
+  dbClusterIdentifier: ${AURORA_DB_CLUSTER_NAME}
+  dbSubnetGroupName: ${RDS_SUBNET_GROUP_NAME}
+  engine: aurora-postgresql
+  engineVersion: "13"
+  publiclyAccessible: false
+EOF
+
+kubectl apply -f asv2-db-instance.yaml
+```
+
+In the `DBCluster` custom resource, you **must** set both the `minCapacity` and
+`maxCapacity` attributes in the `serverlessV2ScalingConfiguration` section,
+otherwise the database cluster will not be created.
```

Comment:
Note that you can inject a stylized notation using the following syntax:

```
{{% hint title="YOUR TITLE HERE" type="info" %}}
YOUR TEXT HERE
{{% /hint %}}
```

If you look at the [overview page](https://aws-controllers-k8s.github.io/community/docs/community/overview/) for our docs, you will see two stylized note blocks, one titled "Choose the ACK container image that is right for you" and the other titled "Be sure to specify a release version". The former uses no `type` value, the latter uses the `type="hint"` value.

Reply:
:+1: